cmd/helm/installer/install.go

```diff
@@ -129,24 +129,23 @@ func generateLabels(labels map[string]string) map[string]string {
 }
 
 // parseNodeSelectors takes a comma delimited list of key=values pairs and returns a map
-func parseNodeSelectors(labels string) map[string]string {
+func parseNodeSelectors(labels string, m map[string]string) map[string]string {
```

Comment:
By the way, [maps are (effectively) pointers](https://dave.cheney.net/tag/maps). I recalled that article too late to include it earlier.